src/coreclr/System.Private.CoreLib/src/System/Runtime/CompilerServices/RuntimeHelpers.CoreCLR.cs

```diff
@@ -416,6 +416,48 @@ public static IntPtr AllocateTypeAssociatedMemory(Type type, int size)
                 }
             }
         }
+
+        /// <summary>
+        /// Create a boxed object of the specified type from the data located at the target reference.
+        /// </summary>
+        /// <param name="target">The target data</param>
+        /// <param name="type">The type of box to create.</param>
+        /// <returns>A boxed object containing the specified data.</returns>
+        /// <exception cref="ArgumentNullException">The specified type handle is <c>null</c>.</exception>
+        /// <exception cref="ArgumentException">The specified type cannot have a boxed instance of itself created.</exception>
+        /// <exception cref="NotSupportedException">The passed in type is a by-ref-like type.</exception>
```

Comment:
Should we say that this is equivalent to IL box instruction, similar how we say it for sizeof - https://github.com/dotnet/runtime/pull/100618/files#diff-80973f0bb86a2e16b0aec3830a9fcf100d16321b91ec68a3e9f737ed2e224d36R431 ?